packages/x-date-pickers/src/CalendarPickerSkeleton/CalendarPickerSkeleton.tsx

```diff
@@ -103,7 +103,7 @@ function CalendarPickerSkeleton(props: CalendarPickerSkeletonProps) {
     name: 'MuiCalendarPickerSkeleton',
   });
 
-  const classes = useUtilityClasses(props);
+  const classes = useUtilityClasses(other);
```

Comment:
Fix to allow passing `classes` down via `defaultProps.classes`

Reply:
Less changes I guess 🤷 
But good point, will update, does not seem to require those exquisite types as well. 🤔